ChattoAdditions/Source/Input/ExpandableTextView.swift

```diff
@@ -110,6 +110,21 @@ open class ExpandableTextView: UITextView {
         }
     }
 
+    override open func closestPosition(to point: CGPoint) -> UITextPosition? {
+        let pointInTextContainer = self.closestPointInTextContainer(to: point)
+        return super.closestPosition(to: pointInTextContainer)
+    }
+
+    override open func closestPosition(to point: CGPoint, within range: UITextRange) -> UITextPosition? {
+        let pointInTextContainer = self.closestPointInTextContainer(to: point)
+        return super.closestPosition(to: pointInTextContainer, within: range)
+    }
+
+    override open func characterRange(at point: CGPoint) -> UITextRange? {
+        let pointInTextContainer = self.closestPointInTextContainer(to: point)
+        return super.characterRange(at: pointInTextContainer)
+    }
+
```

Comment:
I guess, for someone a default behaviour is more preferable.
Could you make it optional (but enabled by default)?

Reply:
Discussed with @AntonPalich, decided not to make it optional for now.